lib/manageiq/appliance_console/prompts.rb

```diff
@@ -140,6 +146,10 @@ def ask_for_new_password(prompt, default: nil, allow_empty: false, retry_limit:
       end
     end
 
+    def ask_for_messaging_password(prompt)
+      ask_for_new_password(prompt, :validation => MESSAGING_PASSWORD_REGEXP, :validation_err => "special characters are not allowed, except for \"$\" and \"*\"")
```

Comment:
Hm maybe `Allowed characters are a-z, A-Z, 0-9, $, and *`

Reply:
thanks, also included `-, _, .,` in the message. Apparently they are not considered to be special characters 😛

Reply:
They're all special in their own way 😀